css/common.css

```diff
@@ -66,7 +66,7 @@ nav {
   display: inline-block;
   height: 60px;
   line-height: 60px;
-  width: 150px;
+  width: 165px;
```

Comment:
Why this?

Reply:
Ah I see, for the extra button. We can remove Learn for now and revert this.

Reply:
It's for the "Projects" button in the navbar. The text doesn't fit while hovering. Removing the "Learn" button will not fix this.

This will be fixed when the Project section is added to the main page and then we can remove the "Projects" button from the navbar.

Reply:
This breaks responsiveness around `940px`. Maybe reduce the font size a bit instead? (We can also increase the threshold for mobile view but let's try to avoid that)

Reply:
This is now resolved in 40f9c92be28b58db9e497acdd0f284ac92022648 by decreasing the font size and letter spacing a bit.